src/item.cpp

```diff
@@ -3965,6 +3984,19 @@ bool item::ammo_consume( int qty ) {
     return false;
 }
 
+itype_id item::ammo_current() const
+{
+    if( is_ammo() ) {
+        return typeId();
```

Comment:
Noted. Incidentally we use `"NULL"` and `"null"` throughout the codebase. Would there be any objection to a separate PR that converts everything to the same and if so which is the preferred form?